### PR TITLE
fix doc example error (missing import)

### PR DESCRIPTION
**Pull request by njthomas**

### CODE REVIEW

File: docs/pages/asking_for_input.rst [hunk 1]
```diff
@@ -165,6 +165,7 @@ Creating a custom style could be done like this:
 .. code:: python
 
     from prompt_toolkit.shortcuts import prompt
+    from prompt_toolkit.styles import Style
```
Comment (jonathanslenders) [inline docs/pages/asking_for_input.rst:168]: Thanks for this! Would it be possible to merge the import with the next line?

Comment (njthomas) [reply docs/pages/asking_for_input.rst]: Yup, updated commit.

Comment (jonathanslenders) [reply docs/pages/asking_for_input.rst]: Thanks for this!

**Comment by jonathanslenders** [comment]
Thank you!

**Pull request merged**